client/src/pages/proverb/ProverbList.js

```diff
@@ -1,37 +1,71 @@
-import React, { useEffect } from 'react'
-import { Container } from 'react-bootstrap'
-import FlexTable from '../../components/FlexTable'
-import { useProverb, useSearch } from './../../redux/hooks'
-import { homepageTableTitle } from './../../helpers/flexTableData'
+import React, { useEffect } from "react";
+import { Container } from "react-bootstrap";
+import FlexTable from "../../components/FlexTable";
+import Pagination from "../../components/Pagination";
+import { useProverb, useSearch, usePagination } from "./../../redux/hooks";
+import { homepageTableTitle } from "./../../helpers/flexTableData";
 
 const ProverbList = () => {
-  const { proverbs, getProverbs } = useProverb()
-  const { filtered, isActive, searchTerm, setSearch } = useSearch()
+  const { proverbs, getProverbs, totalPages } = useProverb();
+  const { filtered, isActive, searchTerm, setSearch } = useSearch();
```

Comment:
I would rename this to be very explicit that it is the filtered list of proverbs

```suggestion
  const { filtered: filteredProverbs, isActive, searchTerm, setSearch } = useSearch();
```

maybe also could change line 9 to:

```
  const { proverbs: allProverbs, getProverbs, totalPages } = useProverb();

```


then in your code below it makes it very clear:

`items={isActive ? filteredProverbs : allProverbs}`